content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,178 @@
-**attributes**.
+In this example, the **T-Shirt** is the product template, and **T-Shirt, S, Blue** is a product
+variant. **Color** and **size** are the attributes. **S** and **Blue** are values.
 
-The above example has a total of 20 different products (5 sizes x 4
-colors). Each one of these products has its own inventory, sales, etc.
+The above example has a total of 20 different product variants (5 sizes x 4 colors). Each one of
+these has its own inventory, sales, etc.
 
-Impact of variants
-==================
-
-- **Barcode**: the code and barcode is associated to a variant, not the
-  template. Every variant may have its own barcode / SKU.
-
-- **Price**: every product variant has its own public price that is
-  computed based on the template price ($20) with an optional extra
-  for every variant (+$3 for color red). However, you can define
-  pricelist rules that apply on the template or the variant.
-
-- **Inventory**: the inventory is managed by product variant. You don't
-  own t-shirts, you only own "T-shirts, S, Red", or "T-Shirts, M,
-  Blue". For information purpose, on the product template form, you
-  get the inventory that is the sum of every variant. (but the
-  actual inventory is computed by variant)
-
-- **Picture**: the picture is related to the variant, every variation
-  of a product may have its own primary picture.
-
-- **Other fields**: most of the other fields belongs to the product
-  template. If you update them, it updates automatically all the
-  variants. (example: Income Account, Taxes)
-
-Should you use variants?
-========================
-
-When should you use variants?
------------------------------
-
-Using variants has the following impacts:
-
-- **eCommerce**: in your online shop, the customer will only see
-  product templates in the catalog page. Once the visitor click on
-  such a product, he will have options to choose amongst the
-  variants (colors, sizes, …)
+Configuration
+=============
 
-- **Manufacturing**: Using variants allows to define only one bill of
-  material for a product template and slight variations for some of
-  the variants. Example: instead of creating a Bill of Material for
-  "T-shirt, Red, S", you create a bill of material for "T-shirt"
-  and add some lines that are specific to the dimension S, and
-  other lines specific to the color Red.
+Activating product variants
+---------------------------
 
-- **Pricing**: The default price of a product is computed using the
-  price of the product template and add the optional extra price on
-  each dimension of the variant. This way, variant prices are
-  easier to maintain since you don't have to set the price for
-  every variant. However, it's possible to create pricelist rules
-  to fix price per variants too.
+To use product variants, head over to the :guilabel:`Sales` app, select
+:menuselection:`Configuration --> Settings`, and enable the :guilabel:`Variants` feature.
```

Comment:
Add `` Finally, click :guilabel:`Save` to apply the setting. `` to the end of the paragraph